tofu/entrypoints/tofuplot.py

```diff
@@ -3,13 +3,20 @@
 # Built-in
 import sys
 import os
-import argparse
+
 
 # Generic
 import matplotlib.pyplot as plt
 plt.switch_backend('Qt5Agg')
 plt.ioff()
 
+
+# import parser dict
+sys.path.insert(1, _TOFUPATH)
```

Comment:
same question, `_TOFUPATH` seems undefined her

Reply:
You are absolutely right @lasofivec , I had missed this and it was fixed by the following PR #405 👍 

Good to see you here :-) !